src/widget/loginscreen.cpp

```diff
@@ -31,6 +31,16 @@
 #include <QMessageBox>
 #include <QDebug>
 
+#ifdef Q_OS_WIN32
+    #include <windows.h>
+#else
+    #include <X11/XKBlib.h>
```

Comment:
This will introduce a new dependency, which will have to be listed in `INSTALLING.md`.
Also new dependencies have to be agreed by our new committee.